### PR TITLE
fix: adds better parsing of decimals for use with small numers and am…

### PR DESCRIPTION
**Pull request by R-K-H**
…ounts

### CODE REVIEW

File: components/Markets/ConditionalMarketCard.tsx [hunk 1]
```diff
@@ -103,6 +103,25 @@ export function ConditionalMarketCard({
   )) * markets.fail.quoteLotSize.toNumber()) / markets.fail.baseLotSize.toNumber(),
   [markets.fail.baseLotSize]);
 
+  const splitDecimals = (value: number) => {
```
Comment (swaggymarie) [inline components/Markets/ConditionalMarketCard.tsx]: splitDecimals name might be misleading what about getDecimalAmount?
can it be moved to lib ?

Comment (swaggymarie) [reply components/Markets/ConditionalMarketCard.tsx]: otherwise good to go

Comment (R-K-H) [reply components/Markets/ConditionalMarketCard.tsx]: Good point, I was just changing it a lot so kept it there, we can and will fix.

Comment (R-K-H) [reply components/Markets/ConditionalMarketCard.tsx]: Fixd

**Pull request merged**